.github/ISSUE_TEMPLATE/BUG.yml

```diff
@@ -42,28 +51,27 @@ body:
         2. Click on '....'
         3. Scroll down to '....'
         4. See error
-      render: bash
     validations:
       required: false
   - type: textarea
     id: screenshot
     attributes:
       label: "Screenshots"
       description: If applicable, add screenshots to help explain your problem.
-      value: |
-        "![SCREENSHOT_DESCRIPTION](SCREENSHOT_LINK.png)"
-      render: bash
+      placeholder: Drag-and-drop images up add them directly or use Markdown to embed external images.
     validations:
       required: false
   - type: textarea
     id: logs
     attributes:
       label: "Logs"
-      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
-      render: bash
+      description: >-
+        Please copy and paste any relevant log output.
+        This will be automatically formatted into code, so no need for backticks.
+      render: text
     validations:
       required: false
-  - type: textarea
```

Comment:
This might contain multiple lines, I would not recommend to change that

Reply:
reverted this in c540895fe8bf7011ea2d4bcb63438dd5f3bda140